src/test/java/pages/LoginPage.java

```diff
@@ -6,7 +6,8 @@
 
 public class LoginPage extends BasePage{
 
-    @FindBy(css="[type='email']")
+    public HomePage login;
+    @FindBy(css = "[type='email']")
```

Comment:
**I recommend:  make all the locators and WebElements private,** because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

`For example`


```
    @FindBy(css = "[type='email']")
    private WebElement emailField;

    @FindBy(css="[type='password']")
    private WebElement passwordField;

    @FindBy(css="[type='submit']")
    private WebElement submitButton;

    @FindBy(css="[data-testid='login-form']")
    private WebElement loginForm;
```